### ISSUE TITLE
Multilingual support

### ISSUE THREAD

**Issue by loretoparisi** [usefulsensors/moonshine#23]
While both tiny and base model achieve impressive performances 💯 in terms of WER on academics datasets when compared to the "standard-de-facto" (e.g. Whisper), in a real world scenery a multi-lingual model would address more use cases.
Fine-tune train code could eventually enable support by the community.

**Comment by evmaki** [comment]
Yes, I think releasing some fine-tuning code would be beneficial. I can add it to our TODO list.

**Comment by bil-ash** [comment]
@evmaki Please provide the code for finetuning, would be beneficial for extending to other languages.

**Comment by zalastone** [comment]
Korean and Mandarin will boost their popularity in ASR use cases . it's great work guys

**Comment by loretoparisi** [comment]
The reason why this (multilingual support) is extremely important can been see from this simple chart related to whisper fine-tune for downstream tasks of transcription (WER) and translation (BLEU) and the correlation to the amount (in hours) of audio transcribed or translated respectively. By example while **Spanish (ES)**  exhibits the Best WER (2.5) for speech recognition it has the Lowest BLEU score (24) for translation, while **German (DE)** has balanced performance (WER: 4, BLEU: 35), or - viceversa **Portuguese (PT)** shows the High BLEU score (39) for translation but a relatively low WER (4) for speech recognition, etc.
<img width="682" alt="Screenshot 2024-12-04 alle 23 45 12" src="https://github.com/user-attachments/assets/3d78e783-4129-484f-9b4d-e56681f6b278">
Approximative data derived from [Robust Speech Recognition via Large-Scale Weak Supervision](https://cdn.openai.com/papers/whisper.pdf)

**Comment by bil-ash** [comment]
@evmaki Still looking forward to the code for fine-tuning on non-English language or for training a new model from scratch on non-English language, since non-English language models have been released 3 months ago.
